packages/graphback-core/src/crud/mappingHelpers.ts

```diff
@@ -62,15 +65,10 @@ export const getInputTypeName = (typeName: string): string => {
   return `${typeName}Input`;
 }
 
+export function getUserModels(schema: GraphQLSchema) {
```

Comment:
[Possible bugs] This method is kinda hard to grasp as we doing completely separate processing here on top of schema. 
Maybe we should simply pass models here